extras/kinesis-asl/src/main/scala/org/apache/spark/streaming/kinesis/KinesisBackedBlockRDD.scala

```diff
@@ -67,13 +74,13 @@ class KinesisBackedBlockRDDPartition(
 private[kinesis]
 class KinesisBackedBlockRDD(
     sc: SparkContext,
-    regionId: String,
-    endpointUrl: String,
-    @transient blockIds: Array[BlockId],
-    @transient arrayOfseqNumberRanges: Array[SequenceNumberRanges],
+    val regionName: String,
+    val endpointUrl: String,
+    @transient override val blockIds: Array[BlockId],
```

Comment:
`override val` is redundant since the implementation in the parent class is same.